src/geometry/collider_impl.rs

```diff
@@ -144,6 +150,123 @@ impl Collider {
         SharedShape::round_triangle(a.into(), b.into(), c.into(), border_radius).into()
     }
 
+    fn ivec_array_from_point_int_array(points: &[IVect]) -> Vec<Point<i32>> {
+        points
+            .iter()
+            .map(|p| {
+                #[cfg(feature = "dim3")]
+                return Point::new(p.x, p.y, p.z);
+                #[cfg(feature = "dim2")]
+                return Point::new(p.x, p.y);
+            })
+            .collect::<Vec<_>>()
+    }
+
+    fn vec_array_from_point_float_array(points: &[Vect]) -> Vec<Point<Real>> {
+        points
+            .iter()
+            .map(|p| {
+                #[cfg(feature = "dim3")]
+                return Point::new(p.x, p.y, p.z);
+                #[cfg(feature = "dim2")]
+                return Point::new(p.x, p.y);
+            })
+            .collect::<Vec<_>>()
+    }
```

Comment:
Should these be `.into()` ? To implement in nalgebra `glam` compat layer ?

The cloning things is a bit frustrating but I'm not sure we have better options here.